operations/tempo-mixin/alerts.libsonnet

```diff
@@ -82,11 +82,29 @@
             },
           },
           {
+            // page immediately if > 2 flushes have failed AND flush retries have failed
+            alert: 'TempoIngesterFlushesFailing',
```

Comment:
**suggestion:** Should we tweak one of the alert names to distinguish between the two?